src/tests/class_tests/openms/source/SignalToNoiseEstimator_test.cpp

```diff
@@ -116,29 +116,29 @@ START_SECTION((virtual ~SignalToNoiseEstimator()))
 END_SECTION
 
 
-START_SECTION((virtual void init(const PeakIterator& it_begin, const PeakIterator& it_end)))
-  TestSignalToNoiseEstimator sne;
-  MSSpectrum spec;
-  sne.init(spec.begin(), spec.end());
-	NOT_TESTABLE
-END_SECTION
+//START_SECTION((virtual void init(const PeakIterator& it_begin, const PeakIterator& it_end)))
+//  TestSignalToNoiseEstimator sne;
+//  MSSpectrum spec;
+//  sne.init(spec);
+//	NOT_TESTABLE
+//END_SECTION
 
-START_SECTION((virtual void init(const Container& c)))
+START_SECTION((virtual void init(const MSSpectrum& c)))
   TestSignalToNoiseEstimator sne;
   MSSpectrum spec;
   sne.init(spec);
 	NOT_TESTABLE
 END_SECTION
 
-START_SECTION((virtual double getSignalToNoise(const PeakIterator& data_point)))
-  // hard to do without implementing computeSTN_ properly
-	NOT_TESTABLE
-END_SECTION
-
-START_SECTION((virtual double getSignalToNoise(const PeakType &data_point)))
+START_SECTION((virtual double getSignalToNoise(const Size index)))
   // hard to do without implementing computeSTN_ properly
```

Comment:
What does "properly" mean? Maybe it can be tested now?

Reply:
```suggestion
  // hard to do without implementing computeSTN_ properly
```
It is just a comment. Nothing between START_SECTION and END_SECTION can be tested.